moneta/db/moneta/schema/auth_user.sql

```diff
@@ -0,0 +1,12 @@
+/*
```

Comment:
add this script to CREATE_DB.sql
create data script for this table, and add him in INSERT_DB.sql

Reply:
What did you mean?)